src/status_im2/navigation/roots.cljs

```diff
@@ -2,18 +2,18 @@
   (:require [quo2.foundations.colors :as colors]
             [react-native.platform :as platform]
             [status-im2.navigation.view :as views]
-            [status-im2.navigation.state :as state]
             [status-im2.common.theme.core :as utils.theme]))
 
 (defn status-bar-options
-  []
-  (let [dark-mode? (if (= @state/root-id :shell-stack) (colors/dark?) (utils.theme/dark-mode?))]
-    (if platform/android?
-      {:navigationBar {:backgroundColor colors/neutral-100}
-       :statusBar     {:backgroundColor :transparent
-                       :style           (if dark-mode? :light :dark)
-                       :drawBehind      true}}
-      {:statusBar {:style (if dark-mode? :light :dark)}})))
+  ([dark-root?]
+   (let [dark-mode? (if dark-root? (colors/dark?) (utils.theme/dark-mode?))]
+     (if platform/android?
+       {:navigationBar {:backgroundColor colors/neutral-100}
+        :statusBar     {:backgroundColor :transparent
+                        :style           (if dark-mode? :light :dark)
+                        :drawBehind      true}}
+       {:statusBar {:style (if dark-mode? :light :dark)}})))
```

Comment:
Is possible to use Kebab Case for the keys here?

Reply:
hmm, I think it's possible but I don't know what will happen with the merge of all statusBar key uses, is it better to do as a follow up/refactor of roots code?

Reply:
Of course we can do it in a separate PR